content/blog/2021-02-04-skywalking-trace-threadpool/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Apache SkyWalking: How to propagate context between threads when using ThreadPoolExecutor"
+date: 2021-02-04
```

Comment:
2 things left.
1. Are you sure, MyWrapper is a good name to deliver for the end user?
2. If (1) is not problem, please update this to the current date, and no block from me.

Reply:
I remove the MyWrapper and process.png.
There is also some little change to make the logic clear.